libraries/raw-rs/camera_data/Sony/DSLR-A100.toml

```diff
@@ -0,0 +1 @@
+camera_to_xyz = [9437, -2812, -774, -8405, 16215, 2291, -709, 596, 7181]
```

Comment:
Can you please make it store these in decimal form?

Reply:
This is done. Note that it is still stored as `u16` in the binary to save space.